bin/disco_elasticsearch.py

```diff
@@ -0,0 +1,101 @@
+                        help='Log in debug level')
+    parser.add_argument("--env", dest="env", help="Environment name", type=str)
+
+    subparsers = parser.add_subparsers(help='Sub-command help')
+
+    parser_list = subparsers.add_parser("list", help="List all ElasticSearch domains")
+    parser_list.set_defaults(mode="list")
+    parser_list.add_argument("--endpoint", dest="endpoint", action='store_const', default=False, const=True,
+                             help="Display AWS-provided endpoint")
+
+    parser_update = subparsers.add_parser("update",
+                                          help="Update an ElasticSearch domain. If no options are provided, "
+                                          "default behavior is to update all ElasticSearch domains found in "
+                                          "the config.")
+    parser_update.set_defaults(mode="update")
+    parser_update.add_argument("--name", dest="name", type=str, action="append",
+                               help="Name of the ElasticSearch domain")
+
+    parser_delete = subparsers.add_parser("delete",
+                                          help="Delete an ElasticSearch domain. If no options are provided, "
+                                          "default behavior is to delete all ElasticSearch domains found in "
+                                          "the config.")
+    parser_delete.set_defaults(mode="delete")
+    parser_delete.add_argument("--name", dest="name", type=str, action="append",
+                               help="Name of the ElasticSearch domain")
+    parser_delete.add_argument("--all", dest="delete_all", action='store_const', default=False, const=True,
+                               help="Delete *all* ElasticSearch domains")
+
+    return parser
+
+
+def run():
+    """Parses command line and dispatches the commands"""
+    parser = get_parser()
+    args = parser.parse_args()
+    configure_logging(args.debug)
+    env = args.env
+    disco_es = DiscoElasticsearch(env)
+    interactive_shell = sys.__stdin__.isatty()
+
+    if args.mode == "list":
+        entries = disco_es.list(include_endpoint=args.endpoint)
+        headers = ["Elastic Search Domain Name", "Internal Name", "Route 53 Endpoint"]
+        format_line = u"{0:<28} {1:<15} {2:<35}"
+        if args.endpoint:
+            format_line += u" {3:<80}"
+            headers.append("Elastic Search Endpoint")
+        print(format_line.format(*headers), file=sys.stderr)
+        for entry in entries:
+            values = [entry["elasticsearch_domain_name"], entry["internal_name"], entry["route_53_endpoint"]]
+            if args.endpoint:
+                values.append(entry["elasticsearch_endpoint"] or u"-")
+            print(format_line.format(*values))
+
+    elif args.mode == "create":
+        if args.name:
+            for name in args.name:
+                disco_es.create(name)
+        else:
+            disco_es.create()
```

Comment:
Dead code for the win!

My actual intention was to only have an 'update' command from the CLI, so no specific create command. This would give elasticsearch the same 'interface' that elasticache and rds expose.

So I'll remove the create mode handling, if there's no objections.